### DISCUSSION TITLE
Does soketi support webhooks delay ?

### DISCUSSION THREAD

**Discussion by LAGGOUNE-Walid** [soketi/pws#854] [Q&A]
The webhoosk delay are detailed here : 
https://pusher.com/docs/channels/server_api/webhooks/#webhook-request-delay

**Comment by wsamoht** [comment]
I don't think so from this open issue - https://github.com/soketi/soketi/issues/830

**Reply by LAGGOUNE-Walid** [reply]
Okay , thanks . It would be great if soketi respects 100% pusher specifications.
